app/views/layouts/maintenance_tasks/application.html.erb

```diff
@@ -80,7 +80,7 @@
             },
             error => location.reload()
           )
-        }, 3000)
+        }, <%= MaintenanceTasks.default_refresh_rate.in_seconds * 1000 %>)
```

Comment:
We should probably set the value in `[data-refresh]`. This will fail the subresource integrity check for the `<script>` tag if a different value is used.

Reply:
Yes, except I was thinking we could reuse the `data-refresh` attribute on the HTML side. And on the Ruby side we don't need two instance variables.
There's also the case of the `tasks#show` view, which is set in this spot: https://github.com/Shopify/maintenance_tasks/blob/03a38d78659888fa3b3a94402c506777db84878a/app/views/maintenance_tasks/tasks/show.html.erb#L45

With your current changes, it refreshes as fast as it can, which is a lot 😂 I guess the parseInt returns NaN and setTimeout interprets this as 0, as soon as possible.
I think we could do without the parsing anyway given [non-number delay values are silently coerced into numbers](https://developer.mozilla.org/en-US/docs/Web/API/Window/setTimeout#non-number_delay_values_are_silently_coerced_into_numbers).